### DISCUSSION TITLE
is it possible to save the parameter when cloning a voice?

### DISCUSSION THREAD

**Discussion by GpuLover** [neonbjb/tortoise-tts#676] [Q&A]
Hi, 

each time I do "python tortoise/do_tts.py --text "some text" --voice myvoice --preset standard" it seems to be recomputing something and output 3 slightly different voices.

is there a way to save the parameters of voice 2 for example in order to reuse it? to get consistent result accros various voice generations

Thanks.

**Comment by CoderCowMoo** [comment]
TorToiSe uses the current time as a seed for each generation, so you're going to have to hardcode a seed value https://github.com/neonbjb/tortoise-tts/blob/1e061bc6752f05bccb59748c8bd7c7fc85d54988/scripts/tortoise_tts.py#L206, here and play around with it to see a generation you like.

There's also a degree of dropout in the layers but I'm unsure if this is enough to significantly effect the generation.



EDIT: My apologies, I neglected to mention https://github.com/neonbjb/tortoise-tts/blob/1e061bc6752f05bccb59748c8bd7c7fc85d54988/Advanced_Usage.md#generating-conditioning-latents-from-voices
